test/python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 base58
 bip32
 GitPython
-pytezos==3.10.2
+pytezos
```

Comment:
Fixing the version prevents errors caused by changes to the library. Why is the version removed here?

Reply:
If it not causing errors , its better to use the latest version. Ex. ragger